helm/nats-operator/values.yaml

```diff
@@ -137,3 +137,12 @@ configReload:
   repository: "connecteverything/nats-server-config-reloader"
   tag: "0.2.2-v1alpha2"
   pullPolicy: "IfNotPresent"
+
+## Prometheus Metrics Exporter
+##
+metrics:
+  enabled: false
+  registry: "docker.io"
+  repository: "ttjiaa/prometheus-nats-exporter"
```

Comment:
Thanks for the contribution, let's use this image instead which is the one that we will be supporting going forward: `synadia/prometheus-nats-exporter:0.1.0`

Reply:
Thanks! I updated the code and the PR description accordingly.